extensions/amp-ad/0.1/amp-ad-3p-impl.js

```diff
@@ -150,6 +195,7 @@ export class AmpAd3PImpl extends AMP.BaseElement {
    * @override
    */
   onLayoutMeasure() {
+    this.lifecycleReporter_.sendPing('adRequestStart');
```

Comment:
`adRequestStart` actually happens inside `layoutCallback`, after creating the iframe